### PR TITLE
Feature: app layout

### PR DESCRIPTION
**Pull request by ireemalqarni**
App layout contains:
- [ ]  app has a layout folder
- [ ]  app has a layout component in the layout folder called MainLayout.tsx
- [ ]  app has layout imported as the only component in the App.tsx

scr4

### CODE REVIEW

File: src/layout/MainLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+function MainLayout() {
+
+  return (
+    <>
+     <h1>Hello World!</h1>
+    </>
+  )
+}
+
+export default MainLayout;
```
Comment (MaymoonaAlBoloshi) [inline src/layout/MainLayout.tsx]: layouts usualy accept a child to render with in their tags
like <MainLayout>
<some component> // children
</MainLayout>

**Review by MaymoonaAlBoloshi — CHANGES_REQUESTED** [review]

File: src/layout/MainLayout.tsx (hunk 1 above)
Comment (ireemalqarni) [reply src/layout/MainLayout.tsx]: changed done
scr2

Comment (ireemalqarni) [reply src/layout/MainLayout.tsx]: Changed done 
scr2

**Review by MaymoonaAlBoloshi — APPROVED** [review]
LGTM

**Pull request merged**